netlify.toml

```diff
@@ -44,6 +44,10 @@
   from = "/scipylib/donations.html"
   to = "/donations/"
 
+[[redirects]]
+  from = "/gethelp"
+  to = "/community/"
```

Comment:
@jarrodmillman can you confirm that I am doing this right? Thanks!

Reply:
I think this is correct, using the preview and adding `gethelp`, the redirection is going through 👍